src/gui/tray/activitylistmodel.cpp

```diff
@@ -310,76 +335,17 @@ void ActivityListModel::startFetchJob()
     job->start();
 }
 
-void ActivityListModel::activitiesReceived(const QJsonDocument &json, int statusCode)
+void ActivityListModel::ingestActivities(const QJsonArray activities)
```

Comment:
`const QJsonArray &activities`
why not passing by const reference ?

Reply:
Now passed by reference